index.html

```diff
@@ -142,53 +182,58 @@ <h2 class="about-sub-head">Let's Connect</h2>
                         <li><a href="#"><img src="icon/git.svg"></a></li>
                     </ul>
                 </div>
-                     <button class="resume-btn">Get My resume</button>
+                   <button class="resume-btn">Get My resume</button>
             </div>
+
+            
             <div class="language-header">
+
                 <h3 class="language-heading">Languages<i class="fa fa-angle-down"></i></i></h3>
```

Comment:
Mobile version (Your design)
![image](https://user-images.githubusercontent.com/78251927/229274627-be49c2cb-a51d-4a56-ba2d-3483aaa6ec0a.png)

![image](https://user-images.githubusercontent.com/78251927/229274748-b268d3ed-b801-410a-8f7d-51261026a5b5.png)


Please, after implementing the desktop design, kindly ensure also that the mobile design still looks like expected using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Reply:
I have observed that the icons are still missing.

![image](https://user-images.githubusercontent.com/61361037/229302181-ef9e5d2f-2413-4776-83a9-00d2c9eb5225.png)
Kindly, try to implement the desktop design, that looks like the provided Figma design.